src/components/side-panel/add-elements-panel/add-elements-panel.tsx

```diff
@@ -0,0 +1,48 @@
+import { Text } from "@lifesg/react-design-system/text";
+import noop from "lodash/noop";
+import { ElementSelectorCard } from "src/components/element-selector-card";
+import { ELEMENTS_CATEGORIES } from "src/data";
+import { EElementType } from "src/schemas";
+import {
+    Category,
+    CategoryHeader,
+    ElementButtonList,
+    Wrapper,
+} from "./add-elements-panel.styles";
+
+export const AddElementsPanel = () => {
+    // =============================================================================
+    // RENDER FUNCTIONS
+    // =============================================================================
+    const renderElementButtons = (elementTypes: EElementType[]) => {
+        return elementTypes.map((elementType) => {
+            return (
+                <li key={EElementType[elementType]}>
```

Comment:
`EElementType[elementType]` is returning undefined as `elementType` is not referencing the key of the enum

Reply:
Yea actually i need not do this. Just `elementType` will do. Erroneous